### PR TITLE
Use python3-style super() calls

### PR DESCRIPTION
**Pull request by Grazfather**
Since we don't use python2 we should be able to clean up the super calls.

Also, we don't lint bad super calls so we can remove the pylint comment.

### CODE REVIEW

File: gef.py [hunk 1]
```diff
@@ -8419,7 +8419,7 @@ class HexdumpQwordCommand(HexdumpCommand):
     _example_ = "{:s} qword $rsp L16 REVERSE".format(_cmdline_)
 
     def __init__(self):
-        super(HexdumpCommand, self).__init__(complete=gdb.COMPLETE_LOCATION) #pylint: disable=bad-super-call
+        super().__init__()
```
Comment (hugsy) [inline gef.py:8422]: Why did you remove the completion class here?
Also `gdb.COMPLETE_SYMBOLS` is here probably more suited than location

File: gef.py [hunk 2]
```diff
@@ -9890,7 +9888,7 @@ class GefCommand(gdb.Command):
     _syntax_  = "{:s} (missing|config|save|restore|set|run)".format(_cmdline_)
 
     def __init__(self):
-        super(GefCommand, self).__init__(GefCommand._cmdline_,
+        super().__init__(GefCommand._cmdline_,
                                          gdb.COMMAND_SUPPORT,
```
Comment (hugsy) [inline gef.py:9892]: Can you tab-align?

**Review by hugsy — CHANGES_REQUESTED** [review]
Globally ok, not sure why you removed the completer for hexdump and patch.
Few stuff to fix

File: gef.py (hunk 1 above)
Comment (Grazfather) [reply gef.py:8422]: the completion is adopted from its super class

File: gef.py [hunk 3]
```diff
@@ -10203,7 +10198,7 @@ class GefSaveCommand(gdb.Command):
     _syntax_  = _cmdline_
 
     def __init__(self, *args, **kwargs):
-        super(GefSaveCommand, self).__init__(GefSaveCommand._cmdline_, gdb.COMMAND_SUPPORT,
+        super().__init__(GefSaveCommand._cmdline_, gdb.COMMAND_SUPPORT,
                                              gdb.COMPLETE_NONE, False)
```
Comment (hugsy) [inline gef.py:10202]: still not align

File: gef.py [hunk 4]
```diff
@@ -10243,7 +10238,7 @@ class GefRestoreCommand(gdb.Command):
     _syntax_  = _cmdline_
 
     def __init__(self, *args, **kwargs):
-        super(GefRestoreCommand, self).__init__(GefRestoreCommand._cmdline_,
+        super().__init__(GefRestoreCommand._cmdline_,
                                                 gdb.COMMAND_SUPPORT,
```
Comment (hugsy) [inline gef.py:10242]: same here
and below

**Review by hugsy — CHANGES_REQUESTED** [review]

**Comment by Grazfather** [comment]
Oops! My mistake.

**Review by hugsy — APPROVED** [review]
